docs/kubectl-plugins.md

```diff
@@ -14,10 +14,23 @@ kubectl open notebook -n my-namespace my-nb-name
 kubectl open notebook -f notebook.yaml
 ```
 
-Install with:
+### Installation
+
+Clone the repo:
+
+```sh
+git clone https://github.com/substratusai/substratus && cd substratus
+```
+
+Install the binary:
 
 ```sh
-# TODO: Install from GitHub release.
-go build ./kubectl/open-notebook && mv open-notebook /usr/local/bin/kubectl-open-notebook
+go build ./kubectl/open-notebook
```

Comment:
TODO(bjb): modify these once I have a clear idea of how the binaries show up in github artifacts